### ISSUE TITLE
fp-auto-height css/scss missing

### ISSUE THREAD

**Issue by jessetaverne** [alvarotrigo/fullPage.js#1496]
Hey there,

Like everyone else here i'll start of with some praise.
The plugin is simply amazing!

Not sure if this is the right place to mention this, as i'm not very into github:

I was going to mention the fp-auto-height function wasn't working properly, but of course, it is.
The reason it didn't out of the box for me was that the required CSS didn't come with the plugin.

As you mentioned at #1263 the auto-height works well with the CSS.

The latest master i pulled was v2.7.1, in which neither the css nor the scss files have the following:

```css
.fp-auto-height.fp-section,
.fp-auto-height .fp-slide,
.fp-auto-height .fp-tableCell{
    height: auto !important;
}
```

They're also still at v2.6.6.

Hope this helps, maybe i overlooked something though - in that case sry, and can you tell me what i missed?

**Comment by alvarotrigo** [comment]
> The latest master i pulled was v2.7.1, in which neither the css nor the scss files have the following:

[It's here](https://github.com/alvarotrigo/fullPage.js/blob/master/jquery.fullPage.css#L204).
I forgot to update from 2.6.6 to 2.6.7.

The CSS and JS versions are currently under the same version number. That's just because I don't usually commit the CSS files unless they suffer from any change. 

I'll take a look at the best approach for the versioning...

**Comment by jessetaverne** [comment]
Thanks,

Probably missed it because it's in the CSS file - it's actually not included in the SCSS file though - which is the one I used.

Not included yet <a href="https://github.com/alvarotrigo/fullPage.js/blob/master/jquery.fullPage.scss#L234">here</a>.

**Comment by alvarotrigo** [comment]
> it's actually not included in the SCSS file though - which is the one I used.

Right. Good point.

**Comment by alvarotrigo** [comment]
Solved in 2.7.2

**Comment by jtlscout** [comment]
I figured out that the reason it doesn't work is because the CDN on the GitHub repository doesn't support fp-auto-height. Here's what does:
<link href="https://rawgit.com/alvarotrigo/fullPage.js/master/jquery.fullPage.css" rel="stylesheet"> <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.js"></script> <script src="https://rawgit.com/alvarotrigo/fullPage.js/master/jquery.fullPage.js"></script>
